symengine/functions.h

```diff
@@ -529,6 +529,7 @@ class Dirichlet_eta : public OneArgFunction
     RCP<const Basic> rewrite_as_zeta() const;
     //! \return Canonicalized zeta
     virtual RCP<const Basic> create(const RCP<const Basic> &arg) const;
+    virtual RCP<const Basic> create(const vec_basic &arg) const;
```

Comment:
This is not needed here as it is defined in OneArgFunction. You can make it visible by adding `using OneArgFunction::create`